src/render.js

```diff
@@ -12,6 +12,7 @@ import { slice } from './util';
  * existing DOM tree rooted at `replaceNode`
  */
 export function render(vnode, parentDom, replaceNode) {
+	setup();
```

Comment:
This would be needed to avoid us reaching into globals while we renderToStream or renderToString